src/olympia/amo/management/commands/compress_assets.py

```diff
@@ -0,0 +1,302 @@
+import hashlib
```

Comment:
All of this jingo_minify dependency-removal / folding in can be uplifted right now and doesn't depend on Django 1.11